### PR TITLE
Submission of translated language documents

### PR DESCRIPTION
**Pull request by 2216923014**

### CODE REVIEW

File: src/主要的/资源/zh.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+lang: Chinese
+version: 0.1
+author: DPOH-VAR
```
Comment (DPOH-VAR) [inline src/主要的/资源/zh.yml:3]: 😄

**Comment by DPOH-VAR** [comment]
Hi!

Can I post your translation file to spigotmc https://www.spigotmc.org/resources/powernbt.9098/
as link to unofficial translation? I will mention you as author of this file.

Give me a link to your zh.yml file on github.

Also, change the author name.

**Comment by 2216923014** [comment]
That's careless , When you enter the path, the input is in Chinese. Forget it,Look at my other request.



---Original---
From: "Andrey ***@***.***&gt;
Date: Mon, Jan 17, 2022 00:29 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [flinbein/PowerNBT] Submission of translated language documents(PR #38)




 
@DPOH-VAR commented on this pull request.
 
 
In src/主要的/资源/zh.yml:
 &gt; @@ -0,0 +1,58 @@ +lang: Chinese +version: 0.1 +author: DPOH-VAR  
😄
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by DPOH-VAR** [comment]
Hi!

**Pull request closed without merging**